AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Benchmark_GPU.py

```diff
@@ -7,13 +7,20 @@
 import logging
 import os
 import psutil
+import sys
 
 import pytest
 
 import editor_python_test_tools.hydra_test_utils as hydra
 from ly_test_tools.benchmark.data_aggregator import BenchmarkDataAggregator
 
 logger = logging.getLogger(__name__)
+WINDOWS = sys.platform.startswith('win')
+
+if not WINDOWS:
+    pytestmark = pytest.mark.skipif(
+        not WINDOWS,
+        reason="TestSuite_Benchmark_GPU.py currently only runs on Windows")
```

Comment:
You should update the description to indicate that the Benchmark GPU is restricted to only run on Windows, instead of being skipped on Linux, since this also prevents it from running on Mac as well

Reply:
updated description in PR